compiler/hie-core/hie-core-in-emacs.md

```diff
@@ -0,0 +1,50 @@
+# How to get `hie-core` working in Emacs
```

Comment:
I'd just fling this as a section in the normal readme for hie-core. I wouldn't have thought of hunting for an emacs specific one.

Reply:
Copy that @neil-da. I'll take care of it Monday. Initial doc. was just to get something going. I'll make it nice and integrate.